package.json

```diff
@@ -953,6 +954,78 @@
       }
     ],
     "debuggers": [
+      {
+        "type": "blazorwasm",
+        "label": "Blazor WebAssembly Debug",
+        "initialConfigurations": [
+          {
+            "type": "blazorwasm",
+            "name": "Launch and Debug Blazor WebAssembly Application",
+            "request": "launch"
+          }
+        ],
+        "configurationAttributes": {
+          "launch": {
+            "properties": {
+              "cwd": {
+                "type": "string",
+                "description": "The directory of the Blazor WebAssembly app, defaults to the workspace folder.",
+                "default": "${workspaceFolder}"
+              },
+              "url": {
+                "type": "string",
+                "description": "The URL of the application",
+                "default": "https://localhost:5001"
+              },
+              "browser": {
+                "type": "string",
+                "description": "The debugging browser to launch (Edge or Chrome)",
+                "default": "chrome",
+                "enum": [
+                  "chrome",
+                  "edge"
+                ]
+              },
+              "trace": {
+                "type": [
+                  "boolean",
+                  "string"
+                ],
+                "default": "true",
+                "enum": [
+                  "verbose",
+                  true
+                ],
+                "description": "If true, verbose logs from JS debugger are sent to log file. If 'verbose', send logs to console."
+              },
+              "hosted": {
+                "type": "boolean",
+                "default": "false",
+                "description": "True if the app is a hosted Blazor WebAssembly app, false otherwise."
+              },
+              "webRoot": {
+                "type": "string",
+                "default": "${workspaceFolder}",
```

Comment:
`webRoot` is only used for hosted apps right? Assuming that's so shouldn't the `default` be associated with the program? Aka `{workspaceFolder}/Server`?

Reply:
`webRoot` is useful if the user runs their Blazor WASM application app at some sub-directory (e.g. http://localhost:5001/dashboard) so it is not specific to hosted scenarios.

I set the default to `workspaceFolder` since that seemed like a generic and un-opinionated default.